src/sys/memfd.rs

```diff
@@ -35,41 +35,54 @@ libc_bitflags!(
         /// See also the hugetlb filesystem in [`memfd_create(2)`].
         ///
         /// [`memfd_create(2)`]: https://man7.org/linux/man-pages/man2/memfd_create.2.html
-        #[cfg(linux_android)]
+        #[cfg(any(linux_android, target_env = "ohos"))]
```

Comment:
`linux_android` means `any(target_os = "linux", target_os = "android")`, and since `ohos` is a Linux libc, so it should be already covered here

Reply:
Sure, so sorry that i didn't note linux_android is different from target_os = linux or target_os = android